plugins/resources/replication/irods_repl_rebalance.cpp

```diff
@@ -199,12 +200,13 @@ namespace {
         genquery_inp_wrapped.get().maxRows = _batch_size;
 
         std::stringstream cond_ss;
+        cond_ss << "IN (";
         for (auto& bun : _bundles) {
             for (auto id : bun) {
-                cond_ss << "= '" << id << "' || ";
+                cond_ss << "'" << id << "',";
             }
         }
-        const std::string cond_str = cond_ss.str().substr(0, cond_ss.str().size()-4);
+        const std::string cond_str = std::string(cond_ss.str().substr(0, cond_ss.str().size()-1)) + ")";
```

Comment:
the `std::string(...)` constructor calls is not needed. `substr()` returns a `std::string`

Reply:
done